org.jreleaser.cli.yml

```diff
@@ -0,0 +1,106 @@
+app-id: org.jreleaser.cli
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+appstream-compose: false
+command: jreleaser 
+finish-args:
+  - --env=PATH=/app/jre/bin:/usr/bin:/app/bin
+  - --env=JAVA_HOME=/app/jre
+  - --share=network
+  - --filesystem=host
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+
+  - name: jreleaser
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 bin/jreleaser /app/bin/jreleaser
```

Comment:
does this use a vulnerable log4j?

Reply:
No. If you look at the manifest file you'll see every single JAR file required by this tool. Log4j is not explicitly listed nor it's found in a repackaged form.